articles/tokens/add-custom-claims.md

```diff
@@ -0,0 +1,22 @@
+---
+description: How to add custom claims to Access Tokens using Rules
+topics:
+  - tokens
+  - access-tokens
+contentType:
+  - how-to
+useCase:
+  - invoke-api
+---
+
+# Add Custom Claims to Access Tokens
+
+You can add custom claims to your Access Token (or [ID Token](/tokens/id-token)) using [Rules](/rules). The claim name must conform to a namespaced format, which basically means adding any non-Auth0 HTTP or HTTPS URL as a prefix. 
```

Comment:
This exists on the ID Tokens page (same exact info):
https://auth0.com/docs/tokens/id-token#add-custom-claims

Maybe this is outside the scope of this work, but seems like we could generalize this doc to be "Add Custom Claim to Tokens" (with text very slightly adjusted), add id-tokens as a topic in the metadata, and link to it from ID Tokens, and call it a day.

Reply:
Good idea.